pkg/controller/command/verifiable/models.go

```diff
@@ -6,13 +6,23 @@ SPDX-License-Identifier: Apache-2.0
 
 package verifiable
 
-import "github.com/hyperledger/aries-framework-go/pkg/store/verifiable"
+import (
+	"encoding/json"
+
+	"github.com/hyperledger/aries-framework-go/pkg/store/verifiable"
+)
 
 // Credential is model for verifiable credential.
 type Credential struct {
 	VerifiableCredential string `json:"verifiableCredential,omitempty"`
 }
 
+// PresentationRequest is model for verifiable presentation request.
+type PresentationRequest struct {
+	VerifiableCredential string          `json:"verifiableCredential,omitempty"`
+	DidDoc               json.RawMessage `json:"doc,omitempty"`
```

Comment:
Add proof options

Reply:
@rolsonquadras ^^

DID ID is enough I think

Reply:
updated